### PR TITLE
Adding special logic for L5 to get OAuth to work

### PR DESCRIPTION
**Pull request by wild-endeavor**

### CODE REVIEW

File: pkg/auth/auth_context.go [hunk 1]
```diff
@@ -172,3 +172,16 @@ func GetOauth2Config(options config.OAuthOptions) (oauth2.Config, error) {
 		},
 	}, nil
 }
+
+func GetL5Oauth2Config(mainConfig *oauth2.Config) oauth2.Config {
```
Comment (katrogan) [inline pkg/auth/auth_context.go]: this is admin public. why are we including l5 specific logic here?

**Comment by katrogan** [comment]
this whole pr shouldn't be in admin public. if we need l5 logic handling can we re-use the existing designated lyft branch? `Lyft-Flyte`

File: pkg/auth/auth_context.go (hunk 1 above)
Comment (wild-endeavor) [reply pkg/auth/auth_context.go]: Can we port this to the Lyft private version?

**Comment by wild-endeavor** [comment]
let me re-open this PR into the lyft specific branch

**Pull request closed without merging**